pkg/logger/logger.go

```diff
@@ -42,6 +42,32 @@ const (
 	HUMAN LogFormat = 1
 )
 
+// LogCode defines the set of  codes that can be set on an Entry
+type LogCode int
+
+const (
+	// NoCode means the entry code hasn't been set
+	NoCode = iota
+	// CloneError means we were unable to successfully clone the resource
+	CloneError
+	// ScanError means there was some issue scanning the cloned resource
+	ScanError
+	// ResourceCleanupError means we couldn't remove the resources that were cloned after a scan
+	ResourceCleanupError
+	// CommandError means there was an error with an external command
+	CommandError
+	// CloneDetail are log entries that are informational
+	CloneDetail
+	// ScanDetail are log entries that are informational
+	ScanDetail
+)
+
+var logCodeNames = [...]string{"NoCode", "CloneError", "ScanError", "ResourceCleanupError", "CommandError"}
```

Comment:
```suggestion
var logCodeNames = [...]string{"NoCode", "CloneError", "ScanError", "ResourceCleanupError", "CommandError", "CloneDetail", "ScanDetail"}
```

Adding these to logCodeNames